doe_dap_dl/dap.py

```diff
@@ -456,8 +456,10 @@ def __download_from_urls(self, urls, path="/var/tmp/", replace=False):
         # TODO: multi-thread this
         for url in urls:
             try:
-                a = url.split("/")
-                filename = a[5].split("?")[0]
+                if 'adc.arm.gov' in url:
```

Comment:
This still feels a bit fragile.  Maybe using https://docs.python.org/3/library/urllib.parse.html would be a bit cleaner?